walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/resolvers/OpenIdURLRequestResolver.kt

```diff
@@ -40,9 +39,7 @@ internal class OpenIdURLRequestResolver(val libraryConfiguration: LibraryConfigu
         if (verifiedIdRequestInput !is VerifiedIdRequestURL) throw UnSupportedVerifiedIdRequestInputException(
             "Provided VerifiedIdRequestInput is not supported."
         )
-        if (libraryConfiguration.isPreviewFeatureEnabled(PreviewFeatureFlags.FEATURE_FLAG_OPENID4VCI_ACCESS_TOKEN))
-            return resolveOpenId4VCIRequest(verifiedIdRequestInput)
-        return OpenIdResolver.getRequest(verifiedIdRequestInput.url.toString())
+        return resolveOpenId4VCIRequest(verifiedIdRequestInput)
```

Comment:
Was removing the feature flag check intentional?

Reply:
Yes, the newer method for resolveOpenID4VCIRequest contains methods for both request format types and support for extensions. The older method calls VC-SDK directly and is resolved to a request without extension involvement.